Assets/Scripts/PopulationController.cs

```diff
@@ -0,0 +1,52 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.EventSystems;
+using Database.Tables;
+using DataStructures;
+
+
+public class PopulationController : MonoBehaviour, IPointerClickHandler
+{    
+    public DatabaseManager _databaseManager;
+    private Canvas _canvas;
+    // Start is called before the first frame update
+    void Start()
+    {
+    }
+
+    // Update is called once per frame
+    void Update()
+    {
+        
+    }
+    private void Awake() {
+        _canvas = transform.Find("Canvas").GetComponent<Canvas>();
+        _canvas.enabled = false;
+    }
+
+    private bool Showing(){
+        return _canvas.enabled;
+    }
+
+    private void Hide(){
+        _canvas.enabled = false;
+    }
+
+    private void Show(){
+        _canvas.enabled = true;
+    }
+
+    public void OnPointerClick(PointerEventData pointerEventData)
+    {
+        if(Showing())Hide();
+        else Show();
+        Debug.Log(name);
+        var sampleList =  _databaseManager.GetSamplesForPopulation(Int32.Parse(name));
```

Comment:
We assume that the name is always a number here. That may not be true in the future. I would add a comment before this line before stating the assumption.